src/ReactDOM.re

```diff
@@ -477,6 +477,91 @@ external unmountComponentAtNode: Dom.element => unit =
 [@mel.module "react-dom"]
 external flushSync: (unit => unit) => unit = "flushSync";
 
+module Experimental = {
+  type preloadOptions;
+
+  [@mel.obj]
```

Comment:
Using `[@mel.obj]` in preloadOptions because wants to use `[@mel.as]` when **_polyvar_** contains preserved keyword and included `-`.